result/bgpv4.yaml

```diff
@@ -24,35 +24,89 @@ components:
           description: >-
             The list of column names that the returned result set will contain.
             If the list is empty then all columns will be returned except for
-            any result_groups.
-            The name of the BGPv4 device cannot be excluded.
+            any result_groups. The name of the BGPv4 peer cannot be excluded.
           type: array
           items:
             type: string
             enum:
               - session_state
-              - routes_advertised
-              - routes_withdrawn
+              - flap_count
+              - routes_sent
+              - routes_received
+              - route_withdraws_sent
+              - route_withdraws_received    
+              - updates_sent
+              - updates_received
+              - opens_sent
+              - opens_received
+              - keepalives_sent
+              - keepalives_received
+              - notifications_sent
+              - notifications_received
 
     Bgpv4.Metric:
       description: >-
-        BGPv4 Router statistics and learned routing information
+        BGPv4 per peer statistics information.
       type: object
       properties:
         name:
           description: >-
-            The name of a configured BGPv4 device.
+            The name of a configured BGPv4 peer.
           type: string
         session_state:
           description: >-
-            Session state as up or down
+            Session state as up or down.
           type: string
           enum: [up, down]
-        routes_advertised:
+        flap_count:
           description: >-
-            Number of advertised routes sent
+            Number of times the session went from Up to Down state.
```

Comment:
Should be rephrased as: Number of times session_state changed from up to down.

Reply:
we can but ideally comments should not dependent on variable/function name.

Reply:
I'm sorry, I didn't understand this part.

I'm actually asking here to refer to actual variable name (`session_state`) and its actual values (`up` or `down`).

Reply:
Every description should be indeppendent of others as far possible.